### PR TITLE
Remove plugin implementation from view and retrieve node instance from LocalComposition

### PR DESCRIPTION
**Pull request by KovalevAndrey**
## Description

Remove plugin implementation from `View`. This will allow to merge `AppyxViewTestRule` and `AppyxParentViewTestRule`

## Check list

- [x] I have updated `CHANGELOG.md` if required.
- [x] I have updated documentation if required.

### CODE REVIEW

**Review by CherryPerry — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Pending changes
 
+- [#197](https://github.com/bumble-tech/appyx/pull/197) – **Breaking change**: `ParentNodeView` does not implement plugin anymore. Node instance is retrieved via LocalComposition.
```
Comment (LachlanMcKee) [inline CHANGELOG.md]: Maybe mention that `AbstractParentNodeView` and `AppyxParentViewTestRule.kt` have been removed?

File: libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNodeView.kt [hunk 2]
```diff
@@ -2,9 +2,17 @@ package com.bumble.appyx.core.node
...
 
-interface ParentNodeView<NavTarget : Any> : NodeView, NodeAware<ParentNode<NavTarget>> {
+interface ParentNodeView<NavTarget : Any> : NodeView {
+
     @Composable
     fun ParentNode<NavTarget>.NodeView(modifier: Modifier)
+
+    @Suppress("UNCHECKED_CAST")
+    @Composable
+    override fun View(modifier: Modifier) {
```
Comment (LachlanMcKee) [inline libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNodeView.kt:16]: is this okay if it gets overriden? or should it be final?

Comment (KovalevAndrey) [reply libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNodeView.kt:16]: Ideally it should be final, the problem is that `ParentNodeVew` is an interface. (requested changes)

Comment (LachlanMcKee) [reply libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNodeView.kt:16]: Maybe add some JavaDoc specifying that overriding this function will cause unexpected behaviours?

**Review by LachlanMcKee — APPROVED** [review]

**Pull request merged**